chia/full_node/mempool.py

```diff
@@ -522,6 +522,7 @@ async def create_bundle_from_mempool_items(
                         unique_additions.extend(spend_data.additions)
                     cost_saving = 0
                 else:
+                    # item gets modified in the following call
```

Comment:
It doesn't always get modified, so perhaps something like this?
```suggestion
                    # Update this item with fast forwarded spends, if it has any.
```